Makefile

```diff
@@ -1,5 +1,6 @@
 build:
-	go get github.com/elastic/elastic-package
+	go get -ldflags "-X github.com/elastic/elastic-package/cmd.CommitHash=`git describe --always --long --dirty` -X github.com/elastic/elastic-package/cmd.BuildTime=`date +%FT%T%z`" \
```

Comment:
This is a nitpick so feel free to ignore.

The one issue I see with using `date +%FT%T%z` is that it generates the date in the timezone of the system where the build is made. So, for example, in my case that's `PDT` right now, corresponding to an offset of `-0700`. This is not a problem for accuracy because of the `%z` at the end of the format string — the offset gets included as well.

However, it might be a bit inconvenient if the user of `elastic-package version` was not in the same timezone as where it was built. WDYT about instead using `date +%s` here to store the number of seconds since the Epoch, UTC. Then in `versionCommandAction` you can convert this to the user's local timezone and display that for the user's convenience.

Reply:
This is good idea :) Fixed.